src/contracts/core/DelegationManager.sol

```diff
@@ -37,17 +38,21 @@ contract DelegationManager is
     // @dev Chain ID at the time of contract deployment
     uint256 internal immutable ORIGINAL_CHAIN_ID;
 
-    // @dev Maximum Value for `stakerOptOutWindowBlocks`. Approximately equivalent to 6 months in blocks.
-    uint256 public constant MAX_STAKER_OPT_OUT_WINDOW_BLOCKS = (180 days) / 12;
+    /// @dev The minimum number of blocks to complete a withdrawal of a strategy. 50400 * 12 seconds = 1 week
+    uint256 public constant LEGACY_MIN_WITHDRAWAL_DELAY_BLOCKS = 50_400;
```

Comment:
any chance renaming this breaks any integrations?

Reply:
great question, we should keep around a view function with the same signature, at the least

Reply:
We previously had a `getWithdrawalDelay(IStrategy[] calldata strategies)` view func to get the minimum delay given a bunch of strategies since we intended to have EIGEN on a different withdrawal delay.
This has been removed though and I'm not even sure if integrators/LRTs were even using this to begin with.

Regardless, adding the legacy view function in a separate PR